src/panels/lovelace/entity-rows/hui-climate-entity-row.ts

```diff
@@ -0,0 +1,69 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../../../components/ha-climate-state.js";
+import "../components/hui-generic-entity-row.js";
+
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+
+class HuiClimateEntityRow extends LitElement implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: EntityConfig;
+
+  static get properties() {
+    return {
+      hass: {},
+      _config: {},
+    };
+  }
+
+  public setConfig(config: EntityConfig): void {
+    if (!config || !config.entity) {
+      throw new Error("Invalid Configuration: 'entity' required");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (
+      !this._config ||
+      !this._config.entity ||
+      !this.hass!.states[this._config.entity]
```

Comment:
You should only chekc for `!this.hass` here. Don't check if entity exist in states as that is already handled by `<hui-generic-entity-row>`